### ISSUE TITLE
Subchapters 2.19 and 19.2 - Readme.rmd over Readme.qmd

### ISSUE THREAD

**Issue by matanhakim** [hadley/r-pkgs#954]
After reading chapter 2 (and following up on chapter 19, seeing that this isn't handled elsewhere) I was wondering why should you use `.rmd` files rather than `.qmd` files.
As a new-ish useR, I try to adopt best practices from the get-go. As I understood from the latest Rstudio::conf, Quarto is seen as the publishing platform of the future. Personally, I use only Quarto and `.qmd` files now, to invest my gained experience in Quarto rather than Rmarkdown.
Having this in mind, when I saw that the Readme file is `.rmd` and not `.qmd` I wondered What is the reason for this.
One plausible explanation was that these `{devtools}` functions, such as `use_readme_rmd()`, were developed before Quarto, or that the `{pkgdown}` system is intended tow work with Rmarkdown and not Quarto.
In any case, I think it is appropriate to consider an explanation for this choice, even as a footnote.

(This book is a joy to read and really motivates me to create my first package!)

**Comment by jennybc** [comment]
The most active discussion of this is in https://github.com/r-lib/usethis/issues/1671.

TL;DR it's easy to imagine that some of the files we currently handle with `.Rmd` inside a package might shift to `.qmd`. But it's early days and too soon for the tooling/book to take a strong position.
